recipe/meta.yaml

```diff
@@ -12,32 +14,48 @@ source:
   - url: https://raw.githubusercontent.com/antlr/antlr4/{{ version }}/LICENSE.txt
     sha256: b1b379fcaf3219593a4c433feb1b35c780bed23fafaae440b1ae2771a9521e3a
 
+
 build:
-  number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
-
-requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-  host:
-    - python
-    - pip
-  run:
-    - python
-
-test:
-  requires:
-    - pip
-  imports:
-    - antlr4
-    - antlr4.atn
-    - antlr4.dfa
-    - antlr4.error
-    - antlr4.tree
-    - antlr4.xpath
-  commands:
-    - pip check
+  number: 1
+
+
+outputs:
+
+  - name: {{ old_name }}
```

Comment:
Old/deprecated name should be the meta package and the new name should have the actual files in it.